packages/web-components/src/components/ic-breadcrumb-group/ic-breadcrumb-group.stories.mdx

```diff
@@ -453,3 +453,73 @@ import BreadcrumbReadme from "../ic-breadcrumb/readme.md";
     `}
   </Story>
 </Canvas>
+
+### Playground
+
+export const defaultArgs = {
+  appearance: "default",
+  backBreadcrumbOnly: false,
+  collapsed: false,
+  current: false,
+  href: "/",
+  pageTitle: "Home",
+  iconSlot: true,
+};
+
+<Canvas>
+  <Story
+    args={defaultArgs}
+    parameters={{ loki: { skip: true } }}
+    argTypes={{
+      appearance: {
+        options: ["default", "dark", "light"],
+        control: {
+          type: "radio",
+        },
+      },
+      iconSlot: {
+        mapping: {
+          true: "icon",
+          false: "",
+        },
+      },
+    }}
+    name="Playground"
+  >
+    {(args) =>
+      html`<ic-breadcrumb-group
```

Comment:
Please can you add a dark background here for the light variant like you've done for the React storybook?